src/io/flutter/logging/FlutterLogView.java

```diff
@@ -39,8 +37,32 @@
 import java.awt.datatransfer.StringSelection;
 import java.awt.event.MouseEvent;
 
+import static com.intellij.ui.SimpleTextAttributes.STYLE_PLAIN;
+
 public class FlutterLogView extends JPanel implements ConsoleView, DataProvider, FlutterLog.Listener {
 
+  private class EntryModel implements FlutterLogTree.EntryModel {
+    boolean color;
+
+    @Override
+    public SimpleTextAttributes style(@Nullable FlutterLogEntry entry, int attributes) {
+      if (color && entry != null) {
```

Comment:
I'm not sure about the java coding style.
But I think we can use earlier return to reduce the deep nested.
Eg
```java
if (entry == null || notEnableLogColor) return;
// Do other things.
```

Reply:
I'm on the fence.  Many argue that having multiple return points in a function is as confusing (if not more). 🤔 

That said, when a function is very short and can be read in a glance I tend to think either approach is fine.

Thanks!